examples/test_tract/src/main.rs

```diff
@@ -0,0 +1,27 @@
+use tract_core::ndarray;
+use tract_core::prelude::*;
+use rand::*;
+
+fn main() -> TractResult<()> {
+    // load the model
+    let model = tract_tensorflow::tensorflow().model_for_path("./my_model.pb")?;
+    println!("Loaded model");
+

```

Comment:
I'd prefer the example to optimise the model here (or after the input in created). The sequence should look like the one in https://github.com/snipsco/tract/tree/master/examples/tensorflow-mobilenet-v2#a-look-at-the-code, with the `set_input_fact` and into_optimized() call.

I just realized the tract-tensorflow crate example is not like that and will fix it promptly.